docs/commands/rpc-protocol.md

```diff
@@ -610,7 +610,7 @@ Parameters for state block:
 * `key` (optional): Instead of using "wallet" & "account" parameters, you can directly pass in a private key.
 * `source` (optional): The block hash of the source of funds for this receive block (the send block that this receive block will pocket).
 * `destination` (optional): The account that the sent funds should be accessible to.
```

Comment:
@dsiganos Can you confirm if this optional `destination` for `block_create` RPC is supposed to be the account public key or the account address `nano_`? That is unclear and may need to be updated.

Reply:
Thanks @dsiganos I added links to the glossary to clarify this